README.md

```diff
@@ -27,37 +27,85 @@
   </ul>
 </details>
+### Key Features <a name="key-features"></a>
+- **[Linters]**
+- **[Webhint]**
+- **[Stylelint]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+### Prerequisites
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
+- IDE(code editor) like: **Vscode**, **Sublime**, etc. 
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/) 
+
+### Setup 
+
+1. Download the **Zip** file or clone the repo with:
+```bash git clone git@github.com:Surafels/Portfolio_Mobile.git ``` 
+2. To access cloned directory run: 
+```bash cd Portfolio_Mobile ``` 
+3. Open it with the live server 
+
+### Install 
+
+```bash npm install ``` 
+
+## Run
+
+Test for Linters For tracking linter errors locally you need to follow these steps: 
+1. For tracking linter errors in HTML files run:
+```bash npx hint . ``` 
+2. For tracking linter errors in CSS or SASS files run:
+```bash npx stylelint "\*_/_.{css,scss}" ``` 
+3. For tracking linter errors in JavaScript files run:
+```bash npx eslint . ``` 
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-- **[mobile portfolio]**
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
-## 💻 Getting Started <a name="getting-started"></a>
+
 ## 👥 Authors <a name="authors"></a>
+
 👤 **Author1**
-- GitHub: [https://github.com/Surafels](https://github.com/githubhandle)
-## 📝 License <a name="license"></a>
-This project is [MIT](./LICENSE) licensed.
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+- GitHub: [@Surafels](https://github.com/Surafels)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+- [ ] **[feature/setub]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please put star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] This link is broken, Kindly fix it before requesting another review. From the files in your directory, I can see the license file named `MIT.md` Kindly rectify this link to point to mit not `LICENSE` which is not present in your directory